restapi/src/main/java/io/stargate/web/docsapi/service/query/filter/operation/BooleanValueFilterOperation.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package io.stargate.web.docsapi.service.query.filter.operation;
+
+/**
+ * Predicate that can compare values against a {@link Boolean} database value.
+ *
+ * @param <I> Type of the filter value to compare to.
+ */
+public interface BooleanValueFilterOperation<I> extends BaseFilterOperation {
```

Comment:
Isn't `I` always `Boolean`?

Reply:
It's not:
- for the in predicate `<I>` is `List<?>`
- for the exists it's `Boolean` but for all operations, so `StringFilterOperation<Boolean>`